backend/capellacollab/users/tokens/routes.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import base64
+
+import fastapi
+from sqlalchemy import orm
+
+from capellacollab.core import credentials, database
+from capellacollab.core.authentication import injectables as auth_injectables
+from capellacollab.users import injectables as user_injectables
+from capellacollab.users import models as users_models
+
+from . import crud
+
+router = fastapi.APIRouter(
+    dependencies=[
+        fastapi.Depends(
+            auth_injectables.RoleVerification(
+                required_role=users_models.Role.USER
+            )
+        )
+    ]
+)
+
+
+@router.get("/{user_id}/token")
+def get_token_for_user(
+    user: users_models.DatabaseUser = fastapi.Depends(
+        user_injectables.get_existing_user
+    ),
+    db: orm.Session = fastapi.Depends(database.get_db),
+):
+    password = credentials.generate_password(32)
+    crud.create_token(db, user.id, password, "new_token")
+    new_token = user.name + ":" + password
+    return base64.b64encode(new_token.encode("ascii"))
```

Comment:
we should return the bare password as capellambse does not use the `basic token` but rather the username and password and encodes them in the request then